chapters/en/chapter13/3.mdx

```diff
@@ -0,0 +1,322 @@
+# Understanding the DeepSeek R1 Paper
+
+This chapter is a crash course paper reading. We will walk through the paper in simple terms, and then we will break down the key concepts and takeaways.
+
+DeepSeek R1 represents a significant advancement in language model training, particularly in developing reasoning capabilities through reinforcement learning. The paper introduces a new reinforcement learning algorithm called Group Relative Policy Optimization (GRPO).
+
+![DeepSeek R1 Overview](https://huggingface.co/reasoning-course/images/resolve/main/grpo/4.png)
+
+In the next chapter, we will build on this knowledge and implement GRPO in practice.
+
+The initial goal of the paper was to explore whether pure reinforcement learning could develop reasoning capabilities without supervised fine-tuning. 
+
+<Tip>
+Up until that point, all the popular LLMs required some supervised fine-tuning, which we explored in [chapter 11](/chapters/en/chapter11/1).
+</Tip>
+
+## The Breakthrough 'Aha' Moment
```

Comment:
maybe we can put a collab notebook/ a space where people can play w/ 1.5B DS R1 model

Space would be better IMO

Reply:
Or Inference Providers - up to you!